fiftyone/core/utils.py

```diff
@@ -19,15 +14,24 @@
 import logging
 import ntpath
 import os
-import posixpath
 import platform
+import posixpath
 import signal
 import struct
 import subprocess
 import timeit
 import types
 import zlib
 
+from base64 import b64decode
```

Comment:
nit: per [style guide](https://github.com/voxel51/fiftyone/blob/develop/STYLE_GUIDE.md#imports), we currently ignore `import`/`from` for grouping purposes.

IE, import blocks are defined solely based on whether it is builtin/third-party/project, not by import strategy used.